prost-types/src/protobuf.rs

```diff
@@ -992,10 +992,10 @@ pub struct Any {
     /// used with implementation specific semantics.
     ///
     #[prost(string, tag="1")]
-    pub type_url: std::string::String,
+    pub type_url: ::alloc::string::String,
```

Comment:
Hmm its possible this should not have the leading `::` and thats why I was getting build failures without `extern crate alloc` in the protobuf crate